RockLib.Messaging.Kafka/StatisticsExtensions.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Reflection;
```

Comment:
Is this actually needed?  I don't see anything that would use this but I could be missing it.

Reply:
It isn't needed, I just forgot to do a "remove unneccessary usings"